front/app/containers/Admin/formBuilder/components/FormBuilderToolbox/index.tsx

```diff
@@ -82,10 +82,11 @@ const FormBuilderToolbox = ({
           fontWeight="normal"
           mb="4px"
           mt="24px"
-          ml="12px"
+          ml="16px"
           variant="h6"
           as="h3"
           color="secondaryText"
+          style={{ textTransform: 'uppercase' }}
```

Comment:
@benfraserdesign I think this is ok (using the textTransform) :) Not sure if we have any languages in the back office which can't be transformed to uppercase though (e.g. arabic?) or where it wouldn't make sense?

Reply:
Arabic has no lower/uppercase, so I think CSS transform just doesn’t do anything